optuna/multi_objective/_hypervolume/base.py

```diff
@@ -24,9 +63,9 @@ def _validate(solution_set: np.ndarray, reference_point: np.ndarray) -> None:
         # Validates that all points in the solution set dominate or equal the reference point.
         if not (solution_set <= reference_point).all():
             raise ValueError(
-                "All solution must dominate or equal the reference point. "
-                "That is, for all solution in the solution_set and the coordinate `i`, "
-                "`solution[i] <= reference_point[i]`."
+                "All points must dominate or equal the reference point. "
+                "That is, for all points in the solution_set and the coordinate `i`, "
+                "`point[i] <= reference_point[i]`."
```

Comment:
The inequality is rendered in italics. Do you add one more pair of backticks similar to the `compute`?

Current
![image](https://user-images.githubusercontent.com/3255979/91440684-ed811c00-e8a9-11ea-81c2-372fc775815b.png)

Add one more pair of backticks:
![image](https://user-images.githubusercontent.com/3255979/91441137-a7788800-e8aa-11ea-9bba-f7d1a4a76f67.png)